composer.json

```
@@ -0,0 +1,31 @@
+{
+    "name": "utopia-php/fetch",
+    "description": "A simple library that provides an interface for making HTTP Requests.",
+    "type": "library",
+    "license": "MIT",
+    "require": {
+        "php": ">=8.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "laravel/pint": "^1.5.0",
+        "phpstan/phpstan": "^1.10"
+    },
+    "scripts": {
+        "test": "phpunit --verbose tests",
+        "check": "./vendor/bin/phpstan analyse --level max src tests",
+        "lint": "pint --test",
+        "format": "pint"
```

```suggestion
        "lint": "./vendor/bin/pint --test --config pint.json",
        "format": "./vendor/bin/pint --config pint.json",
        "check": "./vendor/bin/phpstan analyse --level 8 -c phpstan.neon app src tests",
        "test": "./vendor/bin/phpunit --configuration phpunit.xml --debug"
```

Copied those from Open Runtimes executor, for consistency. Logic should be very similar, if not exactly the same.